SI/Sem.02/Pract.02/Task13/Task13.cpp

```diff
@@ -0,0 +1,32 @@
+#include <iostream>
+
+using namespace std;
+
+int main()
+{
+    int a;
+    cout << "Enter a 4-digit natural number: ";
+    cin >> a;
+    if (a < 1000 || a > 9999)
+    {
+        cout << "Invalid input" << endl;
+        return 1;
+    }
+    else
+    {
+        int a1, a2;
+        
+        a1 = (a/1000)*10 + ((a%1000)%100)%10;
```

Comment:
Refactor line 19 and line 20, there is much simpler solution to the the digit
Line 19 just directly use %10 in the end